contrib/jmx-metrics/src/main/java/io/opentelemetry/contrib/jmxmetrics/JmxMetrics.java

```diff
@@ -75,18 +76,27 @@ private static JmxConfig getConfigFromArgs(final String[] args) {
     if (args.length != 0 && (args.length != 2 || !args[0].equalsIgnoreCase("-config"))) {
       System.out.println(
           "Usage: java io.opentelemetry.contrib.jmxmetrics.JmxMetrics "
-              + "-config <path_to_config.properties>");
+              + "-config <path_to_config.properties or - for stdin>");
       System.exit(1);
     }
 
     Properties props = new Properties();
     if (args.length == 2) {
-      try (InputStream is = new FileInputStream(args[1])) {
-        props.load(is);
-      } catch (IOException e) {
-        System.out.println(
-            "Failed to read config properties file at '" + args[1] + "': " + e.getMessage());
-        System.exit(1);
+      if (args[1].trim().equals("-")) { // parse from stdin
```

Comment:
Nit: a possibility here is to put these blocks into functions with descriptive names that also replace their comments: e.g. `parseFromStdin` and `parseFromPath` below.

Reply:
good idea, updated